frontend/src/entities/jobs/algorithms.tsx

```diff
@@ -52,10 +54,10 @@ export type Algorithm = {
 };
 
 export enum HyperparameterType {
-    CATEGORICAL = 'categorical',
-    CONTINUOUS = 'continuous',
-    INTEGER = 'integer',
-    STATIC = 'static',
+    CATEGORICAL = 'Categorical',
+    CONTINUOUS = 'Continuous',
+    INTEGER = 'Integer',
+    STATIC = 'Static',
```

Comment:
Any particular reason for this change?

Reply:
This is how the console renders it and currently there is inconsistency between the type capitalizations.

<img width="920" alt="Screenshot 2024-03-26 at 12 04 07 PM" src="https://github.com/awslabs/mlspace/assets/46975522/51ab5025-42ba-48bf-bde7-3a664f05355d">
<img width="1037" alt="Screenshot 2024-03-26 at 12 04 29 PM" src="https://github.com/awslabs/mlspace/assets/46975522/aa7663ce-1d0f-42b3-ba98-32ff335bcfa6">